exp/sampler.jl

```diff
@@ -39,6 +39,30 @@ _optimizer_value_type(::Type{Clarabel.Optimizer{T}}) where{T} = T
 _optimizer_value_type(m::MOI.OptimizerWithAttributes) = _optimizer_value_type(m.optimizer_constructor)
 _optimizer_value_type(m::JuMP.AbstractModel) = JuMP.value_type(m)
 
+function set_time_limit(model, config, dataset_name)
```

Comment:
This function will be called each time we build and solve a model, and it triggers a disk I/O operation (reading the reference JSON file). If we have lots of jobs running in parallel, this will strain the I/O and possibly slow down our code.
(for large systems, `case.json` can take a few hundreds of MB).